### ISSUE TITLE
Can't annotate PDFs in Safari

### ISSUE THREAD

**Issue by dwhly** [hypothesis/client#357]
### Steps to reproduce
1. In safari, go here: https://wac.colostate.edu/books/inoue/intro.pdf
2. Click our bookmarklet

### Expected behaviour
PDFjs should be activated
Client should deploy

### Actual behaviour
Nothing happens.

Did this ever work?  I would have assumed so.  But honestly I can't recall.

### Browser/system information

This was taken from a via proxied version of the same document:
1.12.1
User agent:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/603.1.30 (KHTML, like Gecko) Version/10.1 Safari/603.1.30
URL:
https://wac.colostate.edu/books/inoue/intro.pdf
PDF fingerprint:
6bf89714d7287547a0746e0e7feceb8b
Username:
dwhly
Date:
15 Apr 2017 15:32:49 -0700

**Comment by dwhly** [comment]
Ticket: https://hypothesis.zendesk.com/agent/tickets/1101

**Comment by robertknight** [comment]
> Did this ever work? I would have assumed so. But honestly I can't recall.

No. The bookmarklet/client never had any logic in it to redirect the user to Via.

From a little experiment I've just done it appears that the Safari PDF viewer is the native PDF viewer inserted into a trivial HTML wrapper page using `<embed>`. Activating the bookmarklet _does_ add the embed script to the page, although it doesn't do anything. This means that it _might_ be possible to get the client to redirect the user to Via in this case.
